doc/classes/EditorSettings.xml

```diff
@@ -1079,9 +1079,11 @@
 		</member>
 		<member name="run/bottom_panel/action_on_play" type="int" setter="" getter="">
 			The action to execute on the bottom panel when running the project.
+			[b]Note:[/b] This option won't do anything if the bottom panel switching is locked using the button in the bottom-right corner of the bottom panel.
```

Comment:
The button isn't in the bottom-right when using the editor with a right-to-left language where the UI gets mirrored, so it's probably best to avoid using this kind of location adjectives. Not sure what would be a good alternative though.

Reply:
I guess now that the icon is a pin instead of a lock, we can use this without being redundant:
```suggestion
			[b]Note:[/b] This option won't do anything if the bottom panel switching is locked using the pin button in the corner of the bottom panel.
```